resources/clients/react/elwing/src/setupTests.ts

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
```

Comment:
// With jest-dom, you can do things like the following:

Reply:
This comment is generated by `create-react-app`. It's a common sight in React land.

Reply:
Got the 'OK' from Liz to leave this alone.